central/clusterinit/backend/backend_impl.go

```diff
@@ -280,3 +287,25 @@ func (b *backendImpl) ValidateClientCertificate(ctx context.Context, chain []mtl
 
 	return nil
 }
+
+func getStoreContext(ctx context.Context, accessMode storage.Access) context.Context {
+	accessLevels := []storage.Access{accessMode}
+	if accessMode == storage.Access_READ_WRITE_ACCESS {
+		accessLevels = append(accessLevels, storage.Access_READ_ACCESS)
+	}
```

Comment:
Out of scope of this PR, but should we at some point make `Access_READ_WRITE_ACCESS` actually include the read so we don't have to do things like this?

Reply:
That would indeed be good. I fear that a number of tests do test the write-only behaviour of `sac.AllowFixedScopes` with `storage.Access_READ_WRITE_ACCESS` only.
What I attempted in that direction did uncover a larger amount of work, which would be good for the consistency of the `ScopeCheckerCore` implementations, but which I would not pull in the set of PRs at hand.

Reply:
I wonder if we should also question the validity of `write-only`.  Seems like a rare use case for a context to truly be `write-only`.  But thats a problem for another day.